docs/diagrams/CreateBP.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
```

Comment:
if you use [mermaid ](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/) then github will render the picture for you as well, which is very nice, please consider

Reply:
Thank you for letting me know. I will probably will